### PR TITLE
Added buda.com crypto site

### PR DESCRIPTION
**Pull request by julian-alarcon**
Added cryptocurrency buda.com site. Alexa rank is [66,668](https://www.alexa.com/siteinfo/buda.com)

### CODE REVIEW

**Comment by phallobst** [comment]
Please replace the icon with this one:
buda

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -160,6 +160,13 @@ websites:
       - totp
     doc: https://blog.btcpop.co/2017/04/27/signing-security-btcpop-co-account/
 
+  - name: Buda
+    url: https://www.buda.com
+    img: buda.png
+    tfa:
+      - totp
+    doc: https://soporte.buda.com/es/articles/2379080-que-es-la-clave-dinamica-y-como-activarla
```
Comment (phallobst) [inline _data/cryptocurrencies.yml]:
```suggestion
    doc: https://soporte.buda.com/es/articles/2379080
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: _data/cryptocurrencies.yml (hunk 1 above)
Comment (julian-alarcon) [reply _data/cryptocurrencies.yml]: Done!

**Comment by julian-alarcon** [comment]
Done with the suggested changes. Please review it!

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**